### PR TITLE
460 Updates to EAR_REC and EAR_MEMO schemas

### PR DESCRIPTION
**Pull request by mjtravers**
[app#460](https://github.com/fecgov/fecfile-web-app/issues/460)

### CODE REVIEW

**Review by toddlees — CHANGES_REQUESTED** [review]

File: schema/EAR_MEMO.json [hunk 1]
```diff
@@ -2,38 +2,32 @@
     "$schema": "https://json-schema.org/draft-07/schema#",
     "$id": "https://github.com/fecgov/fecfile-validate/blob/main/schema/EAR_MEMO.json",
     "version": "v0.0.0.0",
-    "title": "FEC Earmark Receipt (11a - Memo)",
-    "description": "Earmark Receipt-Memo (11a)",
+    "title": "FEC Other Receipts",
```
Comment (toddlees) [inline schema/EAR_MEMO.json:5]: I think this was a mistake

File: schema/EAR_MEMO.json [hunk 2]
```diff
@@ -2,38 +2,32 @@
...
     "type": "object",
     "required": [
         "form_type",
         "filer_committee_id_number",
         "transaction_type_identifier",
         "transaction_id",
+        "back_reference_tran_id_number",
+        "back_reference_sched_name",
         "entity_type",
-        "contributor_organization_name",
```
Comment (toddlees) [inline schema/EAR_MEMO.json:14]: these fields are supposed to be in EARMARK_MEMO if we're following group G https://docs.google.com/spreadsheets/d/1gdcvYQBFMeCGt-1NfVuaSTsB0sB-dl2N/edit#gid=325373431

**Review by toddlees — CHANGES_REQUESTED** [review]

File: schema/EAR_MEMO.json (hunk 1 above)
Comment (mjtravers) [reply schema/EAR_MEMO.json]: has been corrected

File: schema/EAR_MEMO.json (hunk 2 above)
Comment (mjtravers) [reply schema/EAR_MEMO.json:14]: The JSON file has been synced up wtih the spec spreadsheet which corrected this among other issues

**Review by Elaine-Krauss-TCG — APPROVED** [review]

**Pull request merged**